community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/CallProcedureAcceptanceTest.scala

```diff
@@ -42,9 +42,9 @@ class CallProcedureAcceptanceTest extends ExecutionEngineFunSuite {
     // Then
     result.toList should equal(
       List(
-        Map("label" -> "A"),
-        Map("label" -> "B"),
-        Map("label" -> "C")))
+        Map("labels" -> "A"),
```

Comment:
Why key changed? If key can only point to a single string name of the label than I think it is better to keep it 'label'